app/assets/javascripts/admin/orders/controllers/order_controller.js.coffee

```diff
@@ -24,3 +24,7 @@ angular.module("admin.orders").controller "orderCtrl", ($scope, shops, orderCycl
 
   for shop in $scope.shops
     shop.disabled = !$scope.distributorHasOrderCycles(shop)
+
+  # Hides the split button introduced by spree in the order form
+  #   We only have one stock location in OFN so it's meaningless to split the order between stock locations
+  $('.split-item').hide()
```

Comment:
I started by using angular.element here and it worked on the app and rpsec but karma kept failing. This direct use of jquery works for the app, rpsec and karma.

Reply:
That might not be the best way to hide it. How about CSS display: none?

Reply:
Or de-deface it and just remove the element from the view?

Reply:
de-deface is a long way, this split code is mixed with js code that is working for us, the variant auto complete code and add variant to order with shipment selection. there's this issue that would/could include that: https://github.com/openfoodfoundation/openfoodnetwork/issues/3440

Reply:
re the css, do you mean you think it's better to do .css( "display", "none" )?
it's "roughly" the same: http://api.jquery.com/hide/

Reply:
oh, I think you mean that I should just add a css rule for this, no JS. I'll try that!

Reply:
I have to use some long selector to override sprees css "display". I would keep this css version. what do you think?